CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
 # Bcoin Release Notes & Changelog
 
+## v1.x.x
+
+### Migration
+
+The chain indexing subsystem has been refactored to be more modular and
+flexible.
+
+A migration is required to cleanup the old indexes, if present.
+
+``` bash
+$ ./migrate/chaindb4to5.js ~/.bcoin/chain
+```
+
+**Note**: if enabled, tx and addr indexes will be regenerated by rescanning the
+chain on next startup, this process might take a while. Please take the
+potential downtime in re-indexing into account before upgrading.
```

Comment:
Small wording suggestion: "Please take the downtime required for re-indexing into consideration before upgrading."

Reply:
The downtime isn't potential is it?

Reply:
If you did not have indexing enabled, then you won't have downtime.

Reply:
Do you need to do a migration if you don't have indexing enabled?

Reply:
Yes, it will just bump up the chain version.